gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,7 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.7-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.4-bin.zip
```

Comment:
why are we downgrading Gradle? Does Spring not support `8.7`?

Reply:
good catch, I'll switch it back

Reply:
still 8.4